### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by JayH5**
We can probably just start with a giant (500MB+) `golang:onbuild` image and make it better from there..

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.6-onbuild
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+EXPOSE 3000
+VOLUME /logs
+
+CMD ["go-wrapper", "run", \
```
Comment (smn) [inline Dockerfile:7]: How does it know to run logdriver?

Comment (smn) [inline Dockerfile:7]: @JayH5 ^^^

Comment (JayH5) [inline Dockerfile:7]: @smn: go-wrapper magic: https://github.com/docker-library/golang/blob/3cdd85183c0f3f6608588166410d24260cd8cb2f/1.6/go-wrapper

Comment (JayH5) [inline Dockerfile:7]: Logdriver executable also available just as `app` because it's copied in to `/go/src/app`... Weird golang path stuff :/

Comment (smn) [inline Dockerfile:7]: :+1:

**Comment by smn** [comment]
:+1:

**Pull request merged**